src/coordinator/executor/result.go

```diff
@@ -23,43 +23,71 @@ package executor
 import (
 	"github.com/m3db/m3db/src/coordinator/block"
 	"github.com/m3db/m3db/src/coordinator/parser"
+	
+	"github.com/pkg/errors"
 )
 
 const (
 	// TODO: Get from config
 	channelSize = 100
 )
 
+var (
+	errAborted = errors.New("the query has been aborted")
+)
+
 // Result provides the execution results
 type Result interface {
 	abort(err error)
```

Comment:
Do we need to define these private methods on an interface? How does that work if you're trying to create a struct to satisfy `Result` outside of this package?

Reply:
then i'd export them :) Avoiding exporting things until those are needed. But can do it now if you feel that's better.